# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FindBuoysActionGoal action_goal
FindBuoysActionResult action_result
FindBuoysActionFeedback action_feedback
